tests/phpunit/tests/query/lazyLoadTermMeta.php

```diff
@@ -45,6 +45,7 @@ public static function wpSetUpBeforeClass( WP_UnitTest_Factory $factory ) {
 	 * @covers ::wp_queue_posts_for_term_meta_lazyload
 	 */
 	public function test_wp_queue_posts_for_term_meta_lazyload() {
+		$this->reset_lazyload_queue();
```

Comment:
This is needed for the unit test to pass.

Reply:
Odd, I suspect it means that the queue should be reset in the setup rather than tear down. A problem for another day.